File: app/src/main/java/com/ckw/zfsoft/ckwapparchitecture/modules/secondmodule/CupFragment.java
package com.ckw.zfsoft.ckwapparchitecture.modules.secondmodule;

import android.content.Intent;
import android.os.Build;
import android.os.Bundle;
import android.os.Handler;
import android.support.v4.app.ActivityOptionsCompat;
import android.view.View;
import android.widget.Button;

import com.ckw.zfsoft.ckwapparchitecture.R;
import com.ckw.zfsoft.ckwapparchitecture.base.BaseFragment;
import com.ckw.zfsoft.ckwapparchitecture.di.ActivityScoped;
import com.ckw.zfsoft.ckwapparchitecture.modules.firstmodule.photoview.BigPictureActivity;
import com.ckw.zfsoft.ckwapparchitecture.utils.ToastUtils;
import com.github.chrisbanes.photoview.PhotoView;

import javax.inject.Inject;

import butterknife.BindView;

/**
 * Created by ckw
 * on 2017/12/27.
 * 图片显示
 */
@ActivityScoped
public class CupFragment extends BaseFragment {


    @Inject
    public CupFragment(){
    }

    
    @Override
    public void initPresenter() {

    }

    @Override
    protected int getLayoutResID() {
        return R.layout.fragment_cup;
    }

    @Override
    protected void initVariables() {

    }

    @Override
    protected void handleBundle(Bundle bundle) {

    }



    @Override
    protected void operateViews(View view) {

    }

    @Override
    protected void initListener() {




    }




}

File: app/src/main/java/com/ckw/zfsoft/ckwapparchitecture/modules/secondmodule/CupFragment.kt
package com.ckw.zfsoft.ckwapparchitecture.modules.secondmodule

import android.os.Bundle
import android.view.View
import com.ckw.zfsoft.ckwapparchitecture.R
import com.ckw.zfsoft.ckwapparchitecture.base.BaseFragment

/**
 * Created by ckw
 * on 2017/12/13.
 */
class CupFragment: BaseFragment() {

    fun newInstance(): CupFragment {
        val args = Bundle()
        val fragment = CupFragment()
        fragment.arguments = args
        return fragment
    }

    override fun getLayoutResID(): Int {
        return R.layout.fragment_cup
    }

    override fun initVariables() {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun handleBundle(bundle: Bundle?) {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun initViews(view: View?) {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun initListener() {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun initPresenter() {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }
}